### ISSUE TITLE
Add "Fork me on Github"

### ISSUE THREAD

**Issue by indexzero** [nodejitsu/browsenpm.org#33]
We should make it clear that this is Open Source on all pages :smile:

**Comment by alexindigo** [comment]
There are many places where nodejitsu specific parts are hard coded, so it makes it somewhat hard to use as open source product, more like fork and hack on your own thing.

Example:

[browsenpm.org:/pages/package.js#L56](https://github.com/nodejitsu/browsenpm.org/blob/master/pages/package.js#L56) -> [packages-pagelet:/index.js#L69](https://github.com/nodejitsu/packages-pagelet/blob/master/index.js#L69)

Thank you.

**Comment by 3rd-Eden** [comment]
We could just make this configurable using a configuration file.

**Comment by jcrugzz** [comment]
@alexindigo does the registry passed in from the config not work? when a pagelet is extended, it overrides the defaults. Our nodejitsu mirror is just set as the default in the pagelet.

[reference](https://github.com/nodejitsu/browsenpm.org/blob/master/pages/package.js#L43-L46)

**Comment by alexindigo** [comment]
@jcrugzz Actual `Registry.mirrors` stays intact – https://github.com/3rd-Eden/npmjs/blob/master/index.js#L37-L42

**Comment by alexindigo** [comment]
@3rd-Eden Yep, it would help and another problem with dependencies of dependencies that include nodejitsu specific templates, hard to modify things without a lot of forking.
https://github.com/nodejitsu/contour/blob/master/index.js#L11

**Comment by 3rd-Eden** [comment]
It's hard to modify templates and designs of a website without forking anyways. That is kinda out of the scope of this project. If you want to use your own styling or templates you can easily build that your self with the various of pagelets that this site is build-upon. But if there are things that would prevent from running this against a different registry etc, I would see it as a bug but as far as I know, it should all be controlled by the configuration file.
